pages/organizations/[oid]/reuses.vue

```diff
@@ -3,59 +3,14 @@
     <h2 class="!text-sm !mb-2.5">
       {{ $t(`Réutilisations de l'organisation`) }}
     </h2>
-    <LoadingBlock
-      v-if="organization.metrics.reuses"
-      :status
-    >
-      <div class="grid grid-cols-1 lg:grid-cols-3 gap-6">
-        <Card
-          v-for="reuse in reuses.data"
-          :key="reuse.id"
-          :reuse
-        />
-      </div>
-      <Pagination
-        :total-results="reuses.total"
-        :page="reuses.page"
-        :page-size="reuses.page_size"
-        :link="getLink"
-      />
-    </LoadingBlock>
-    <div
-      v-else
-      class="flex flex-col items-center lg:pt-12"
-    >
-      <NuxtImg
-        src="/illustrations/reuse.svg"
-        width="137"
-        height="104"
-      />
-      <p class="mt-4 mb-5 font-bold text-lg">
-        {{ $t(`Cette organisation n'a pas encore publié de réutilisations.`) }}
-      </p>
-      <BrandedButton
-        color="secondary"
-        :href="config.public.reuseGuideUrl"
-      >
-        {{ $t(`Qu'est-ce qu'une réutilisation ?`) }}
-      </BrandedButton>
-    </div>
+    <ReusesListFromOrganization :organization="props.organization" />
```

Comment:
```suggestion
    <ReusesListFromOrganization :organization />
```

`props` is not necessary in template, also if your variable and props have the same name, you can use this shorter syntax